src/lib/features.json

```diff
@@ -2,131 +2,136 @@
   {
     "name": "unloadChunkByEmptyChunk",
     "description": "Chunk unloading is done by sending an empty chunk",
-    "versions": ["1.8"]
+    "versions": ["1.8", "1.8.1", "1.8.2", "1.8.3", "1.8.4", "1.8.5", "1.8.6", "1.8.7", "1.8.8"]
```

Comment:
instead of doing that, what do you think about migrating this file to minecraft data support version which can handle ranges https://github.com/PrismarineJS/minecraft-data/blob/master/data/pc/common/features.json ?

Reply:
Sure thing.